buildenv/jenkins/JenkinsfileBase

```diff
@@ -26,6 +75,30 @@ def test() {
 					CUSTOMIZED_SDK_URL = ''
 				}
 				env.SPEC = "${SPEC}"
+				env.BUILD_LIST = "${getBuildList()}"
+				echo "env.BUILD_LIST is ${env.BUILD_LIST}"
+				
+				//download JCK test materials if BUILD_LIST is jck
+				if ( env.BUILD_LIST == "jck" ){
+					if (fileExists("$WORKSPACE/jck/")) {
```

Comment:
I would prefer  relative path, which would be consistent with other code. Though absolute path may be a better idea?

Reply:
The absolute path will be pass to JCK_ROOT, which is used within Ant build.xml. relative path will cause build failure.

Reply:
I mean if (fileExists("$WORKSPACE/jck/) and dir("$WORKSPACE/jck/